desktop/app/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+const path = require( 'path' );
```

Comment:
Can be reduced to

```
module.exports = {
	env: {
		node: true,
	},
	rules: {
		'import/no-nodejs-modules': 'off',
	},
};
```

Plus adding an exception `// eslint-disable-next-line no-restricted-modules` in desktop/app/lib/crash-tracker/index.js